scripts/dream.py

```diff
@@ -167,6 +177,23 @@ def main_loop(t2i, outdir, parser, infile):
             if not os.path.exists(opt.outdir):
                 os.makedirs(opt.outdir)
             current_outdir = opt.outdir
+        elif prompt_as_dir:
+            if prompt_as_dir == 'simple':
+                # this line will sanitize the entire prompt as one folder
+                subdir = path_filter.sub('_', opt.prompt)[:name_max].rstrip(' .')
+            else:
```

Comment:
This should error if the value isn't `simple` or `y`, lest someone try to put `-p` before the start of their prompt and get confused why it's missing the first word.

Reply:
I'm honestly not a fan of this system cause it could lead to exactly what you said, it was mostly copied over from the previous PR.